.github/workflows/rust.yml

```diff
@@ -0,0 +1,34 @@
+name: rust
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ '*' ]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Install Rust 1.61.0
+      uses: actions-rs/toolchain@v1
+      with:
+          toolchain: 1.61.0
+          override: true
+          components: rustfmt, clippy
+    - uses: actions/checkout@v3
+    - name: Python3 Build
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10.8'
```

Comment:
I thought we were using 3.8.x as reference? @Juan-M-V I seem to recall you mentioned this not too long ago.

Reply:
Last monday I tried to setup the environment with 3.8.x and it fails with an error (print is not defined). But it works with Python 3.10.8.

Reply:
I think @entropidelic found a fix for that.

Reply:
Yes! with the fix merged python version was changed to 3.9 (matching cairo-rs workflow)